docs/debugging/print_breakpoint.md

```diff
@@ -177,6 +177,10 @@ Why? Under the hood, the compiler gets a functional representation of the staged
 
 To preserve the original order of `jax.debug.print`s as written in your Python function, you can use `jax.debug.print(..., ordered=True)`, which will ensure the relative order of prints is preserved. But using `ordered=True` will raise an error under `jax.pmap` and other JAX transformations involving parallelism, since ordering can't be guaranteed under parallel execution.
 
+#### Computation perturbation
+
+Adding `jax.debug.print` or `jax.debug.breakpoint` statements will change the computation that XLA is asked to compile. This can potentially result in numeric discrepancies compared to the same code without debug statements. Keep this in mind when debugging numerical issues, as the act of adding debug statements might affect the behavior you're trying to investigate.
```

Comment:
It might be worth explicitly noting that the reason for this change is that XLA might perform different fusions when compiling.